pkg/server/cloud/aws/instances.go

```diff
@@ -267,17 +283,20 @@ func bootImageSpecToDescribeImagesInput(spec cloud.BootImageSpec) *ec2.DescribeI
 	return input
 }
 
-func getRootDeviceVolumeSizeAndType(blockDevices []*ec2.BlockDeviceMapping, rootDeviceName string) (int32, string) {
-	var rootDiskSize int32
-	volumeType := "gp2"
+func getRootDeviceVolumeSpecs(blockDevices []*ec2.BlockDeviceMapping, rootDeviceName string) EbsSpecs {
+	specs := EbsSpecs{VolumeType: "gp2"}
 	for _, blockDevice := range blockDevices {
 		if aws.StringValue(blockDevice.DeviceName) == rootDeviceName && blockDevice.Ebs != nil {
-			rootDiskSize = int32(aws.Int64Value(blockDevice.Ebs.VolumeSize))
-			volumeType = aws.StringValue(blockDevice.Ebs.VolumeType)
+			specs.VolumeSize = int32(aws.Int64Value(blockDevice.Ebs.VolumeSize))
+			specs.VolumeType = aws.StringValue(blockDevice.Ebs.VolumeType)
+			specs.Iops = aws.Int64Value(blockDevice.Ebs.Iops)
```

Comment:
`Iops` is optional  for gp3:

    	// This parameter is required for io1 and io2 volumes. The default for gp3 volumes
	// is 3,000 IOPS. This parameter is not supported for gp2, st1, sc1, or standard

So it should be a pointer in `Image`. Otherwise we'd set it to `0` in `getBlockDeviceMapping()` if it's not specified here.